<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadioGui</class>
 <widget class="QMainWindow" name="RadioGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RadioGui</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="navigationButtonsWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayoutNavigation">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnPlayingPage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>44</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius: 5px;
	border: 3px solid rgb(72, 126, 176);
	color: rgb(72, 126, 176);
	background-color: rgb(0, 0, 0);
	padding: 5px 5px 5px 5px;
	outline: none;
}

QPushButton:checked {
	border-radius: 5px;
	border: 3px solid rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(72, 126, 176);
	padding: 5px 5px 5px 5px;
	outline: none;
}</string>
         </property>
         <property name="icon">
          <iconset resource="RadioGui.qrc">
           <normaloff>:/Resources/Resources/music.png</normaloff>:/Resources/Resources/music.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSelectSourcePage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>44</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius: 5px;
	border: 3px solid rgb(72, 126, 176);
	color: rgb(72, 126, 176);
	background-color: rgb(0, 0, 0);
	padding: 5px 5px 5px 5px;
	outline: none;
}

QPushButton:checked {
	border-radius: 5px;
	border: 3px solid rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(72, 126, 176);
	padding: 5px 5px 5px 5px;
	outline: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="RadioGui.qrc">
           <normaloff>:/Resources/Resources/layers.png</normaloff>:/Resources/Resources/layers.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSettingsPage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>44</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius: 5px;
	border: 3px solid rgb(72, 126, 176);
	color: rgb(72, 126, 176);
	background-color: rgb(0, 0, 0);
	padding: 5px 5px 5px 5px;
	outline: none;
}

QPushButton:checked {
	border-radius: 5px;
	border: 3px solid rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(72, 126, 176);
	padding: 5px 5px 5px 5px;
	outline: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="RadioGui.qrc">
           <normaloff>:/Resources/Resources/settings.png</normaloff>:/Resources/Resources/settings.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>QPushButton {
	border-radius: 15px;
	border: 5px solid rgb(72, 126, 176);
	color: rgb(72, 126, 176);
	background-color: rgb(0, 0, 0);
	padding: 5px 5px 5px 5px;
}

QPushButton:checked {
	border-radius: 15px;
	border: 5px solid rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(72, 126, 176);
	padding: 5px 5px 5px 5px;
}</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="pagePlaying">
       <layout class="QVBoxLayout" name="verticalLayoutInformation" stretch="1,0,0,0">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblStation">
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelInfo1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: white; 
	background-color: rgb(0, 0, 0);
	padding: 1px 1px 1px 1px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelInfo2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: white; 
	background-color: rgb(0, 0, 0);
	padding: 1px 1px 1px 1px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStartStop">
          <property name="font">
           <font>
            <pointsize>23</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-radius: 15px;
	border: 3px solid rgb(72, 126, 176);
	color: rgb(72, 126, 176);
	background-color: rgb(0, 0, 0);
	padding: 5px 5px 5px 5px;
	outline: none;
}

QPushButton:checked {
	border-radius: 15px;
	border: 3px solid rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(72, 126, 176);
	padding: 5px 5px 5px 5px;
	outline: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="RadioGui.qrc">
            <normaloff>:/Resources/Resources/play-button.png</normaloff>:/Resources/Resources/play-button.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSelectSource">
       <layout class="QGridLayout" name="gridLayoutButton">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="btn1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>19</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-radius: 15px;
	border: 3px solid rgb(72, 126, 176);
	color: rgb(72, 126, 176);
	background-color: rgb(0, 0, 0);
	padding: 5px 5px 5px 5px;
	outline: none;
}

QPushButton:checked {
	border-radius: 5px;
	border: 3px solid rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(72, 126, 176);
	padding: 5px 5px 5px 5px;
	outline: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>110</width>
            <height>40</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btn5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>19</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-radius: 15px;
	border: 3px solid rgb(72, 126, 176);
	color: rgb(72, 126, 176);
	background-color: rgb(0, 0, 0);
	padding: 5px 5px 5px 5px;
	outline: none;
}

QPushButton:checked {
	border-radius: 5px;
	border: 3px solid rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(72, 126, 176);
	padding: 5px 5px 5px 5px;
	outline: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>110</width>
            <height>40</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btn4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>19</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-radius: 15px;
	border: 3px solid rgb(72, 126, 176);
	color: rgb(72, 126, 176);
	background-color: rgb(0, 0, 0);
	padding: 5px 5px 5px 5px;
	outline: none;
}

QPushButton:checked {
	border-radius: 5px;
	border: 3px solid rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(72, 126, 176);
	padding: 5px 5px 5px 5px;
	outline: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>110</width>
            <height>40</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>19</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-radius: 15px;
	border: 3px solid rgb(72, 126, 176);
	color: rgb(72, 126, 176);
	background-color: rgb(0, 0, 0);
	padding: 5px 5px 5px 5px;
	outline: none;
}

QPushButton:checked {
	border-radius: 5px;
	border: 3px solid rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(72, 126, 176);
	padding: 5px 5px 5px 5px;
	outline: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>110</width>
            <height>40</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btn3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>19</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-radius: 15px;
	border: 3px solid rgb(72, 126, 176);
	color: rgb(72, 126, 176);
	background-color: rgb(0, 0, 0);
	padding: 5px 5px 5px 5px;
	outline: none;
}

QPushButton:checked {
	border-radius: 5px;
	border: 3px solid rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(72, 126, 176);
	padding: 5px 5px 5px 5px;
	outline: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>110</width>
            <height>40</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btn6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>19</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-radius: 15px;
	border: 3px solid rgb(72, 126, 176);
	color: rgb(72, 126, 176);
	background-color: rgb(0, 0, 0);
	padding: 5px 5px 5px 5px;
	outline: none;
}

QPushButton:checked {
	border-radius: 5px;
	border: 3px solid rgb(255,255,255);
	color: rgb(255,255,255);
	background-color: rgb(72, 126, 176);
	padding: 5px 5px 5px 5px;
	outline: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>110</width>
            <height>40</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSettings">
       <layout class="QVBoxLayout" name="verticalLayoutSettings">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutVolume" stretch="0,1">
          <item>
           <widget class="QLabel" name="labelVolume">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	color: white;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="RadioGui.qrc">:/Resources/Resources/speakers.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderVolume">
            <property name="font">
             <font>
              <pointsize>23</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QSlider {
	border-radius: 5px;
	border: 2px solid rgb(72, 126, 176);
	color: rgb(72, 126, 176);
	background-color: rgb(0, 0, 0);
	padding: 5px 5px 5px 5px;
	outline: none;
}
</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacerSettings">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RadioGui.qrc"/>
 </resources>
 <connections/>
</ui>
